packages/client/components/TopBarNotifications.tsx

```diff
@@ -21,7 +21,7 @@ interface Props {
 
 const TopBarNotifications = (props: Props) => {
   const {viewer} = props
-  const {notifications} = viewer || {notifications: {edges: []}}
+  const notifications = viewer?.notifications || {edges: []}
```

Comment:
any ideas why viewer would be null?

Reply:
The viewer can be null on the initial render before the query is finished but I'm not sure why the viewer can exist but notifications can be null / undefined